sdk/storage/Azure.Storage.Webjobs.Extensions.Blobs/src/Azure.WebJobs.Extensions.Storage.Blobs.csproj

```diff
@@ -13,8 +13,9 @@
 
   <ItemGroup>
     <Compile Include="$(AzureStorageWebjobsExtensionSharedSources)\**\*.cs" Link="Shared\%(RecursiveDir)\%(Filename)%(Extension)" />
+    <Compile Include="..\..\..\extensions\Microsoft.Azure.WebJobs.Extensions.Clients\src\Shared\*.cs" Link="Shared\%(RecursiveDir)\%(Filename)%(Extension)" />
```

Comment:
could you create a "variable" to capture this path in one place and use in both blobs and queues extension?
also, would that "Shared" get any internal structure? might be worth using `/**/`